payments-core/src/main/java/com/stripe/android/model/PaymentIntent.kt

```diff
@@ -454,5 +459,8 @@ constructor(
                 PaymentIntentJsonParser().parse(it)
             }
         }
+
+        const val CARD = "card"
+        const val REQUIRE_CVC_RECOLLECTION = "require_cvc_recollection"
```

Comment:
Can we make these `internal`? We shouldn't expose them as part of the public API.

Reply:
Updated